### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by ebilling** [uber-go/mock#210]
With the merge last week of https://github.com/uber-go/mock/pull/207 from @tulzke, I was hoping that you would bump the release version.  

Getting latest still builds the Dec 2023 release of v0.4.0.  

Please bump to v0.4.1 so that my team can start using mockgen from main

**Comment by tulzke** [comment]
> Please bump to v0.4.1

I think that v0.5.0 would be more correct because this PR contains new features.

**Comment by stanleyrh** [comment]
@r-hang @sywhang are either of you able to cut a release for this? :pray:

**Comment by tchung1118** [comment]
Update about the release: we ran into an issue while testing tip of main against our internal Go monorepo. We will have to include another change to mockgen to resolve the issue before we can proceed with this release.

**Comment by ebilling** [comment]
Is there an ETA or another issue associated that we can watch?

**Comment by JacobOaks** [comment]
Once #214 merges and we confirm it within our internal monorepo, the release should be unblocked.

**Comment by tchung1118** [comment]
v0.5.0 was tagged and released.
